MultiServer.py

```diff
@@ -235,6 +238,9 @@ def item_names_for_game(self, game: str) -> typing.Optional[typing.Dict[str, int
     def location_names_for_game(self, game: str) -> typing.Optional[typing.Dict[str, int]]:
         return self.gamespackage[game]["location_name_to_id"] if game in self.gamespackage else None
 
+    def checksum(self, game: str) -> typing.Optional[typing.Dict[str, str]]:
+        return self.gamespackage[game]["checksum"] if game in self.gamespackage else None
```

Comment:
This method seems unused and shouldn't its typing be `typing.Optional[str]`?

Reply:
Copy and paste gone wrong. Removed.